tests/installation/validation/ibft.pm

```diff
@@ -79,18 +79,20 @@ sub ibft_validation {
     # Enabling iBFT autoconfiguration for the interfaces should be done in initrd
     if (is_sle('<15-sp4')) {
         assert_script_run 'grep -e rd.iscsi.ibft=1 -e rd.iscsi.firmware=1 /var/log/YaST2/mkinitrd.log';
+        assert_script_run 'ip a | grep -i ibft';
     }
     elsif (is_sle('>=15-sp4') && is_sle('<16')) {
         # In recent products yast2-bootloader calls dracut instead of mkinitrd, so the logs differ
         assert_script_run 'zgrep -e rd.iscsi.ibft=1 -e rd.iscsi.firmware=1 /var/log/YaST2/y2log*';
+        assert_script_run 'ip a | grep -i ibft';
     }
     else {
         # SLE16 uses agama, so search the general logs of agama
         assert_script_run "zgrep -a -e rd.iscsi.ibft=1 -e rd.iscsi.firmware=1 /var/log/agama-installation/logs.tar.gz";
+        assert_script_run 'ip a | grep -i ens';
```

Comment:
a comment explaining the change of the interface name based on comments in the bug.

Reply:
Done